src/components/molecules/card/card.scss

```
@@ -0,0 +1,60 @@
+@use "../../variables" as *;
+
+.card {
+  overflow: hidden;
+  position: relative;
+  border: none;
+  figure {
+    margin: 0;
+    img {
+      z-index: 1;
+      transition: 0.5s ease;
+      overflow: hidden;
+      @media screen and (max-width: $breakpoint-md) {
+        transition: none;
+      }
+    }
+    &:hover {
+      img {
+        transform: scale(1.2);
+        @media screen and (max-width: $breakpoint-md) {
+          transform: none;
+          object-fit: contain;
+        }
+      }
+    }
+  }
+  .two {
+    z-index: 2;
+    position: absolute;
+    font-weight: 700;
+    left: 6px;
+    font-size: 20px;
```

```suggestion
    font-size: $fontsize-l;
```
fontsizeに関しては変数を使いましょう！